plugin.video.livestream/addon.xml

```diff
@@ -1,23 +1,20 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.livestream" name="Livestream" version="2016.3.29" provider-name="eracknaphobia">
+<addon id="plugin.video.livestream" name="Livestream" version="2017.4.3" provider-name="eracknaphobia">
   <requires>
-   <import addon="xbmc.python" version="2.14.0"/>   
-   <import addon="script.module.simplejson" version="3.3.0"/>    
+   <import addon="xbmc.python" version="2.19.0"/>      
   </requires>
   <extension point="xbmc.python.pluginsource" library="main.py">
         <provides>video</provides>
   </extension>  
   <extension point="xbmc.addon.metadata">    
     <platform>all</platform>
-    <summary lang="en">Livestream.com Videos</summary>
-    <description lang="en">Livestream, formerly known as Mogulus, is a live streaming video platform that allows users to view and broadcast video content using a camera and a computer through the internet.</description>
-    <disclaimer lang="en"></disclaimer>
+    <summary lang="en_GB">Livestream.com Videos</summary>
```

Comment:
I am not sure this will work. Have you tested it? Don't forget the lowest version you're targeting is Helix